tests/test_nc_to_mmd.py

```diff
@@ -1557,9 +1590,9 @@ def test_platform_with_gcmd_vocabulary(self):
                        check_only=True)
         ncin = Dataset(md.netcdf_file)
         value = md.get_platforms(mmd_yaml['platform'], ncin)
-        self.assertEqual(value[0]['short_name'], 'Sentinel-1B')
         self.assertEqual(value[0]['long_name'], 'Sentinel-1B')
-        self.assertEqual(value[0]['instrument']['long_name'], 'Synthetic Aperture Radar (C-band)')
+        self.assertEqual(value[0]['instrument']['long_name'], 'Synthetic Aperture Radar')
+        self.assertEqual(value[0]['instrument']['short_name'], 'C-band')
```

Comment:
should be "SAR-C"

Reply:
in the netCDF file we have 
```
:instrument = "Earth Remote Sensing Instruments > Active Remote Sensing > Imaging Radars > SAR > Synthetic Aperture Radar (C-band)" ;
```
so in this case as we have 1 set of () whatever is inside those gets interpreted as the short name, I do not know how to prevent this agnostically, this is why I thought that it was good not to have () in the long_name...
at the moment 
`:instrument = "Earth Remote Sensing Instruments > Active Remote Sensing > Imaging Radars > SAR > Synthetic Aperture Radar (C-band)(SAR-C)" ;` would get parsed correctly 
and `:instrument = "Earth Remote Sensing Instruments > Active Remote Sensing > Imaging Radars > SAR > Synthetic Aperture Radar(SAR-C)" ;` too 
but with the above the short_name ends up being (C-band)....